src/main/webapp/jsp/instructorCourseEnroll.jsp

```diff
@@ -52,91 +68,82 @@
-        <div class="more-info-point-details">
-          If you are not sure about the format of the spreadsheet,
-          <a id="spreadsheet-link" href="https://docs.google.com/spreadsheets/d/1ZThkUnOW9CSM6a9nDWzchh0UOsPU2s9eyVWb5LG5iMk/edit?usp=sharing">here</a> is a sample file.
-        </div>
-      </li>
-      <li>
-        <span class="more-info-point-title">Column headings</span>
-        <div class="more-info-point-details">
-          The column order is not important.
-          Column headings are <u>not</u> case sensitive. e.g. Team, TEAM, team are all acceptable.
-        </div>
-      </li>
-      <li>
-        <span class="more-info-point-title">Columns to include</span>
+        <span class="more-info-point-title">Spreadsheet Information</span>
         <div class="more-info-point-details">
           <ul>
-            <li class="more-info-column-info">
-              <samp>Section</samp> [Compulsory for courses having more than 100 students]: Section name/ID
-            </li>
-            <li class="more-info-column-info">
-              <samp>Team</samp> [Compulsory]: Team name/ID
-              <div class="more-info-point-details">
-                <ul>
-                  <li class="more-info-email-details">
-                    Team must be unique within a course. A team cannot be in 2 different sections.
-                  </li>
-                  <li class="more-info-email-details">
-                    If you do not have teams in your course, use “N/A” as the team name for all students.
-                  </li>
-                </ul>
-              </div>
-            </li>
-            <li class="more-info-column-info">
-              <samp>Name</samp> [Compulsory]: Student name
-            </li>
-            <li class="more-info-column-info">
-              <samp>Email</samp> [Compulsory]: The email address used to contact the student.<br>
-              <div class="more-info-point-details">
-                <ul>
-                  <li class="more-info-email-details">
-                    This need not be a Gmail address.
-                  </li>
-                  <li class="more-info-email-details">
-                    It should be unique for each student.
-                    If two students are given the same email, they will be considered the same student.
-                  </li>
-                </ul>
-              </div>
-            </li>
-            <li class="more-info-column-info">
-              <samp>Comments</samp> [Optional]: Any other information you want to record about a student.
+            <li>
+              If you have student data in a spreadsheet, simply copy the relevant cell-range from your spreadsheet and
+              paste into the table above.
+              <a id="spreadsheet-link"
+                 href="https://docs.google.com/spreadsheets/d/1ZThkUnOW9CSM6a9nDWzchh0UOsPU2s9eyVWb5LG5iMk/edit?usp=sharing">Here</a>
```

Comment:
We can fix this problem for good now: instead of linking to external Google doc, we can just create a small table in this very page itself.

Reply:
Is this ok? @wkurniawan07 

![image](https://user-images.githubusercontent.com/25320782/40766046-e298de08-64e0-11e8-9cc9-25449b896d2e.png)